cmip6_downscaling/data/observations.py

```diff
@@ -35,6 +35,8 @@ def open_era5(variables: Union[str, List[str]], time_period: slice) -> xr.Datase
     years = range(int(time_period.start), int(time_period.stop) + 1)
 
     ds = xr.concat([cat.era5(year=year).to_dask()[variables] for year in years], dim='time')
+    ds = lon_to_180(ds)
```

Comment:
@norlandrhagen @jhamman adding this function here (and in cmip.py) means that we're calling this function twice for the bcsd workflow. I think this is fine but want to flag

Reply:
i ended up picking up this change in the analysis PR - so now it's gone but that makes sense!